audit-log-api/src/handlers/createAuditLogEvent.ts

```diff
@@ -18,6 +18,14 @@ export default async function createAuditLogEvent(event: APIGatewayProxyEvent):
     })
   }
 
+  if (request.messageId === "") {
```

Comment:
I think this write to stdout logic should move to `record-event` lambda. It should write to stdout before sending a request to the API. The reason is that it's something specific for event handler.

Reply:
ok, will look into it. Thank you